grizli/model.py

```diff
@@ -1431,7 +1434,10 @@ def __init__(self, sci=None, err=None, dq=None,
                 raise KeyError(msg)
 
             instrument = h['INSTRUME']
-            filter = utils.get_hst_filter(h, filter_only=True)
+            if instrument in ['NIRISS']:
```

Comment:
I think this isn't necessary now since `get_hst_filter(filter_only=True)` handles NIRISS and returns the `'FILTER'` keyword.

Reply:
Resolved here: 099acf4aecbcdc887c0980dccc530cedd6d2420c